From 6d9e52e1f162295576fd30a3dadeff7f0ddae06a Mon Sep 17 00:00:00 2001
From: Julien Schueller <schueller@phimeca.com>
Date: Fri, 26 Sep 2025 09:51:48 +0200
Subject: [PATCH] CMake: Pin boostrapping sources

---
 OMCompiler/Compiler/boot/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OMCompiler/Compiler/boot/CMakeLists.txt b/OMCompiler/Compiler/boot/CMakeLists.txt
index 750d3ce1c4b..32124549440 100644
--- a/OMCompiler/Compiler/boot/CMakeLists.txt
+++ b/OMCompiler/Compiler/boot/CMakeLists.txt
@@ -7,7 +7,7 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/bomc/sources.tar.gz")
 else()
 #download and unpack the sources
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bomc/)
-file(DOWNLOAD https://github.com/OpenModelica/OMBootstrapping/archive/refs/heads/master.tar.gz
+file(DOWNLOAD https://github.com/OpenModelica/OMBootstrapping/archive/04d16f7.tar.gz
      ${CMAKE_CURRENT_SOURCE_DIR}/bomc/sources.tar.gz
      SHOW_PROGRESS
      STATUS status
