diff --git a/OMCompiler/SimulationRuntime/ParModelica/auto/om_pm_model.cpp b/OMCompiler/SimulationRuntime/ParModelica/auto/om_pm_model.cpp
index 443cddcad8..138bb58bbb 100644
--- a/OMCompiler/SimulationRuntime/ParModelica/auto/om_pm_model.cpp
+++ b/OMCompiler/SimulationRuntime/ParModelica/auto/om_pm_model.cpp
@@ -39,6 +39,7 @@
 #include "om_pm_model.hpp"
 
 #include <cstring>
+#include <fstream>
 // #include <pugixml.hpp>
 
 #include "json.hpp"
